spec/lib/payment_calculator/ecf/contract/uplift_payment_calculations_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe PaymentCalculator::ECF::Contract::UpliftPaymentCalculations do
+  let(:dummy_class) { Class.new { include PaymentCalculator::ECF::Contract::UpliftPaymentCalculations } }
+  let(:call_off_contract) { create(:call_off_contract) }
+  let(:calculator) { dummy_class.new({ contract: call_off_contract }) }
+
+  context "when condition" do
```

Comment:
hehe, maybe we don't need this context if it doesn't say anything meaningful :)

Reply:
Indeed.